js/assessments/passiveVoiceAssessment.js

```diff
@@ -0,0 +1,68 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var fixFloatingPoint = require( "../helpers/fixFloatingPoint.js" );
+
+/**
+ * Calculates the result based on the number of sentences and passives.
+ * @param {object} passiveVoice The object containing the number of sentences and passives
+ * @param {object} i18n The object used for translations.
+ * @returns {{score: number, text}} resultobject with score and text.
+ */
+var calculatePassiveVoiceResult = function( passiveVoice, i18n ) {
+	var percentage = ( passiveVoice.passives / passiveVoice.total ) * 100;
+	percentage = fixFloatingPoint( percentage );
+	var recommendedValue = 10;
+	var score = 9 - Math.max( Math.min( ( 6 / 10 ) * ( percentage - 6.7 ), 6 ), 0 );
```

Comment:
Missing documentation for this formula.
Also, I've seen the `( 6 / 10 )` being written as `0.6` in other assessments. Perhaps this could be extracted into something like a value object, considering this entire formula seems to be used more often.